src/libevent/src/evutil_rand.c

```diff
@@ -195,8 +195,12 @@ evutil_secure_rng_get_bytes(void *buf, size_t n)
 void
 evutil_secure_rng_add_bytes(const char *buf, size_t n)
 {
+#ifdef __APPLE__
+	arc4random_stir();
```

Comment:
Is it a good idea to ignore the contents of ``buf``?

Reply:
It is fine.

The compiler says arc4random_addrandom is deprecated, use arc4random_stir.
Apple's copy of open source code shows arc4random_addrandom as a no-op if not VARIANT_STATIC.

https://opensource.apple.com/source/Libc/Libc-1272.200.26/gen/FreeBSD/arc4random.c.auto.html

Stir allegedly used to use arc4random_addrandom.
Seems like it actually uses arc4_addrandom in the code above.

https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/arc4random.3.html

Reply:
src/libevent/src/evutil_rand.c:201:2: error: 'arc4random_addrandom' is deprecated: first deprecated in macOS 10.12 - use
      arc4random_stir [-Werror,-Wdeprecated-declarations]
        arc4random_addrandom((unsigned char*)buf,
        ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include/stdlib.h:265:7: note: 
      'arc4random_addrandom' has been explicitly marked deprecated here
void     arc4random_addrandom(unsigned char * /*dat*/, int /*datlen*/)
         ^
1 error generated.

Reply:
No warning if target older macOS.
But I think calling a no-op function is worse.